tools/downloader/src/open_model_zoo/model_tools/converter.py

```diff
@@ -120,13 +144,26 @@ def main():
         requested_precisions = set(args.precisions.split(','))
         unknown_precisions = requested_precisions - _common.KNOWN_PRECISIONS
         if unknown_precisions:
+            try:
+                telemetry.send_event('md', 'converter_error', str({'error': 'Unknown precisions specified.', 'model': 'none'}))
+                telemetry.end_session('md')
+                telemetry.force_shutdown(1.0)
+            except NameError:
+                pass
             sys.exit('Unknown precisions specified: {}.'.format(', '.join(sorted(unknown_precisions))))
 
     models = _common.load_models_from_args(parser, args)
 
     output_dir = args.download_dir if args.output_dir is None else args.output_dir
 
     def convert(reporter, model):
+        try:
+            telemetry = tm.Telemetry()
+            telemetry.send_event('md', 'converter_model_name', model.name)
+            telemetry.send_event('md', 'converter_framework', model.framework)
```

Comment:
Not sure this is the right time to report this - ideally we should report all models that the user requested at the _beginning_ of the session, so that if the converter is terminated in the middle, we still know what the user _wanted_ to convert.

Reply:
Moved to the beginning